src/router/trie-router/node.ts

```diff
@@ -198,10 +198,13 @@ export class Node<T> {
 
       curNodes = tempNodes
     }
-    const results = handlerSets.sort((a, b) => {
-      return a.score - b.score
-    })
 
-    return [results.map(({ handler, params }) => [handler, params] as [T, Params])]
+    if (handlerSets.length > 1) {
+      handlerSets.sort((a, b) => {
+        return a.score - b.score
+      })
+    }
+
```

Comment:
Can you write a test to cover this line which Codecov is pointing?

Reply:
Okay, I'll check